tests/com.espressif.idf.ui.test/target/classes/default-test.properties

```diff
@@ -0,0 +1,25 @@
+# Default build wait before the timeout exception is thrown by the test
+default.project.build.wait=800000
+
+# Default project copy delay before the timeout exception is thrown
+default.project.copy.wait=7000
+
+# Default paths for esp env configs (These may need to be updated as per Automation requirements and the env being used)
+
+#Github Actions Path
+default.env.esp.idf.path=dependencies/idf-tools
+default.env.esp.git.path=git
+default.env.esp.python.version=3.9
+default.env.esp.python.path=python3.9
+default.env.esp.idf.download.path=/dependencies/esp-idf-download
+
+
+#Developer Path
+#default.env.esp.idf.path=C:\\Users\\aliaz\\Documents\\DevTools\\ESP-IDF\\esp-idf-v4.2.1
+#default.env.esp.git.path=C:\\Program Files\\Git\\bin\\git.exe
+#default.env.esp.python.version=3.9
+#default.env.esp.python.path=C:\\Program Files\\Python39\\python.exe
+#default.env.esp.idf.download.path=C:\\esp-idf-download
```

Comment:
Could we remove commented code?

Reply:
Kept these paths for the ease of developers